editor.planx.uk/src/ui/shared/SelectMultiple.tsx

```diff
@@ -104,9 +112,14 @@ export const CustomCheckbox = styled("span")(({ theme }) => ({
 }));
 
 export function SelectMultiple<T>(props: Props<T>) {
+  // MUI doesn't pass the Autocomplete value along to the TextField automatically
+  const isSelectEmpty = !props.value?.length;
+  const placeholder = isSelectEmpty ? props.placeholder : undefined
+
   return (
     <FormControl sx={{ display: "flex", flexDirection: "column" }}>
       <StyledAutocomplete<T, true, true, false, "div">
+        sx={{ mt: props.label ? 2 : 0 }}
```

Comment:
Handling this within the styled component is surprisingly complex from a type POV! Managing it here feels like a sensible compromise 👍